### ISSUE TITLE
New users are not very well guided after registration

### ISSUE THREAD

**Issue by trang** [Tatoeba/tatoeba2#2112]
## Problem

Currently, after registration, new members are redirected to the homepage when are shown the following message:

![Screenshot_2020-01-30 Tatoeba Collection of sentences and translations](https://user-images.githubusercontent.com/221850/73484047-e55b0f80-43a0-11ea-8265-74191ca22926.png)

After that, it can be difficult for them to figure out what exactly they should or could do.

## Some history

We used to redirect them to the [Help](https://tatoeba.org/eng/help) page but this page is too verbose and very few people really read such pages as the first thing after they register on website. That is why this redirection was replaced by a redirection to the homepage, with more a compact welcome message. 

## Suggestion

Instead of telling people to add languages to their profiles, the welcome message could be:

> Welcome to Tatoeba!
>
> To start things off, we encourage you to read our Quick Start guide. If you want to read it later, you will find the link in the footer of the website.
> 
> We hope you'll enjoy your time here with us!

If the user doesn't feel like reading, they can just skip it. But otherwise, they can have a look at the guide and get a better idea of how Tatoeba works.

Additionally, we can send a private message with some general tips and a reminder of the link to the Quick Start guide in case the user didn't fully read the welcome message. The idea of sending a private message to new users was [suggested by Pandaa on the Wall](https://tatoeba.org/eng/wall/show_message/34059#message_34059).

**Comment by ckjpn** [comment]
# Some Ideas
These are a few ideas I think may help. It's not a complete list, but something others can read to inspire other ideas.

## Make the Quick Start Guide and Rules and Guidelines Easier to Find
-  Putting a link to these in the top navigation menu would make them move obvious.
- The way the links are now, at the bottom of the page, means many people likely don't notice them.

## Keep the Quick Start Guide Brief and in Easy English
- Make the main Quick Start Guide with the main points new members immediately need to know very short and easy to read. This will make it more likely that it gets read.
- The purpose of a quick start guide is to give people what they need to get started. Things that are likely intuitive shouldn't really be explained here, since the longer a guide gets, the less likely people will read it and benefit from it.
- Make this easy to scan with clear headings and possibly in list format for the main points under these headings.
- Make sure that the most important things we think that new members should know are at the top of the page. (We want complete sentences. We want natural-sounding sentences that native speakers would actually use. The best way to help is by contributing in your strongest language, translating sentences by native speakers of the language you are translating from. Or, whatever TRANG and others think are things we want members to do.)
- Our current quick start guide has grown longer and more wordy as time has gone on, which makes it less likely that people read it completely, or at all. It would likely be useful to go back to a briefer, less wordy quick start guide.
- If we make the main version is in brief and in easy English, many members, even those not good at English, would be able to read it, and it would also make it more likely that we can find people to translate it.

## In Addition to the Quick Start Guide have another guide that is more detailed.
https://en.wiki.tatoeba.org/articles/show/quick-start-guide
- This one, as it currently is, would be a good place to start for this.
- It could also be used to make the more cut-down, brief quick start guide to replace the existing one.

### Guides

#### Rules and Guidelines
https://en.wiki.tatoeba.org/articles/show/guidelines

### How To Guides
Some of these are already here. https://en.wiki.tatoeba.org/articles/show/main

Put these in a list, sorted roughly by importance to a new visitor to the site, putting things that visitors can do without registering and logging in first, followed by things a contributor can do, followed by things advanced contributors can do, followed by things corpus maintainers can do.

How To Guides fall into these categories.
- How to do things that are not intuitive. Things that often need further explanation.
- How to do things that are intuitive to most people. These are things that may need further explanation for some people, or things that have parts in addition the the intuitive parts that people may not notice.

### How to Videos
Once the current re-design of the website if complete and the look of the site becomes stable,  creating step-by-step videos explaining how to do things would be helpful.  I'd suggest doing these in English on YouTube, using the setting to allow anyone to add subtitles. These should be short videos, each focused on one thing. They should be put in a playlist on YouTube, allowing people to watch them all non-stop without needing to click to each one.

## Notes on the Current Quick Start Guide.

### https://en.wiki.tatoeba.org/articles/show/quick-start
- This started out with a few main guidelines that I harvested from TRANG's blog and I had hosted on my own web server.  Sysko imported it into the wiki.
- I put links to the source of each guideline, so it was clear that these were TRANG's guidelines and not something that I created, though I think I re-worded some into easier-to-read English and perhaps shorter sentences. I don't really remember.
- I put the main guideline sentences as sentences in the corpus and linked to them, so they could be translated easily by members.

### How This One Could Be Improved
- Remove things that are not absolutely of immediate need to the first-time visitor.
- And then link to a "How To" guide.

## Make the wiki pages part of the main site.
- Use the same top navigation menu for both tatoeba.org and wiki.tatoeba.org, so people can seamlessly move between the two.
- The way it is now can be a bit confusing for a new member, or even old members, who can get lost and not easily get back to where they were.
- Perhaps, even make the quick start guide and the rules and guidelines tatoeba.org pages, not wiki.tatoeba.org pages.  Use the wiki pages to develop and suggest improvements, but have the pages visitors read as part of the main website. This would not only make it easier for visitors, but would make it so these pages don't get edited by just anybody and make sure they are what TRANG really wants them to be.

## Sending a Brief Welcoming Email Message

### Here is an example of what I send to new members that I notice contributing.

Subject Line:  Japanese to French - Here are some good ways to find sentences to translate.

Welcome to the project.

Here are some good ways to find sentences to translate.

http://study.aitech.ac.jp/tatoeba/translate/links.php?f=jpn&t=fra&u=TRANG

This is a personalized "dashboard" of useful links that you may want to bookmark for future use.

Using the first link on the left side of the page is a one good way to find sentences to translate.

I think you may need to be logged in as “remember me” on tatoeba.org for this to work correctly.


◼ Other Useful Links 

Quick Start Guide
http://en.wiki.tatoeba.org/articles/show/quick-start
(Note that we encourage members to contribute in their own native languages.)

Rules and Guidelines
https://en.wiki.tatoeba.org/articles/show/guidelines 


### Another possibility

Welcome to the project.

Here are some useful links.

Quick Start Guide
http://en.wiki.tatoeba.org/articles/show/quick-start

Rules and Guidelines
https://en.wiki.tatoeba.org/articles/show/guidelines 

### Notes
- Brief and with links for further information would be good.
- Possibly include more links.  (How to add translations, etc.)

**Comment by RyckRichards** [comment]
Actually, we're thinking about the rules too much, I think. Of course they are important but they will eventually read (or get notified) that we have a Quick Start Guide and a FAQ.

Suggestion: On many websites there is a "step-by-step profile creation". I mean, after you answer to a question to proceed in order to create /validate their profile

Example

Please, set up your native language(s). After that, click "next"
(People will have to browse their native language(s) to go to the next page
[next page]
Tell us the languages you're interested in and how fluent you are. You can add more later if you want
[next page]
Add a profile picture
[next page]
Tell us a little bit about yourself (SO the user will fulfill the bio)
[next page]

That's it. You've set up Tatoeba! Please contact us whenever you need :)

**Comment by ckjpn** [comment]
Just brainstorming off of RyckRichards's idea....

Perhaps we could add one more question to the registration page under "what's your native language?" along with a message saying you can add other languages later.

**What's your strongest non-native language? (You can add other languages later in your profile.)**

Doing so would allow a welcome message to include something like the following for someone whose native language was Portuguese and next strongest language was Spanish. This would give a new member a couple of personally useful links.

Show a random selection of Spanish sentences with Portuguese translations.
https://tatoeba.org/sentences/search?trans_filter=limit&trans_link=direct&from=spa&to=por&trans_to=por&sort=random

Show a random selection of Spanish sentences that do not yet have direct Portuguese translations.
https://tatoeba.org/sentences/search?query=&from=spa&to=por&trans_filter=exclude&trans_to=por&sort=random

Or, perhaps tatoeba.org could set up some kind of dashboard of useful links for members like this that requires a native language, another language and optionally a username.

http://study.aitech.ac.jp/tatoeba/translate/links.php?f=spa&t=por&u=Ricardo14

**Comment by jiru** [comment]
To anyone wanting to work on this issue: I strongly encourage you to read the [UX tests](https://en.wiki.tatoeba.org/articles/show/ux-tests) I did, and if possible, to help performing new UX tests. The more we have, the better.

Also note that only adding languages to one’s profile can also be difficult for new users as I mentioned in #1916.

**Comment by jiru** [comment]
I second @RyckRichards’ opinion about the superiority of step-by-step guidance over manuals. I plan to work on this based on the data I gathered in my UX tests. There are two things to consider here: profile filling and website guidance.

About profile filling, my idea is to redirect to a kind of a tutorial instead of the home page with a welcome message. Make a step-by-step thing basically showing the edit profile form and then the add language form (after rearranging it in #1916).

About website guidance, ideally this shouldn’t be necessary if the website is well designed and self-explicit. But it could be another step like "what would you like to do now?" → Add new original sentences, Translate other members’ sentences, Proofread others members’ sentences, Find native speakers of X, Find members learning Y, Ask the community about something… Each item would point users to the right page. But what’s the most important in my opinion is to work on these individual pages to make them more newcomer-friendly. We could put a introductory message explaining what the page is about, with an "I got it" button. This way the information is given to the user at the right moment and within context. This is what makes this approach superior to keeping information in separate guides. To give you an example, we already have such message on the "Adopt sentences" page on the right pane.

I am also considering leading users through an "add your first sentence" and "add your first translation" step after registration.

**Comment by trang** [comment]
Step-by-step tutorial is definitely better than a manual, but more costly to implement. I actually wanted to suggest that as a Google Summer of Code project but I was a bit concerned if we have a UI stable enough to start something like that.

Could you describe a bit more what the tutorial would look like? Just to know if I should start to think about the responsive redesign of the pages involved in your tutorial.

**Comment by jiru** [comment]
I was thinking about re-using existing pages. Something like showing the contents of `/user/edit_profile` with a title like "Please tell us a bit about yourself" and a different header/footer allowing to go back or skip some parts of the tutorial. As for the "add language" step, it depends on #1916 so I’m gonna start by working on that anyway.

We could as well break it down into more simple steps as RyckRichards suggested, but I’m afraid it may annoying to have to many steps.

**Comment by RyckRichards** [comment]
Maybe we can have "less" steps than I suggested. I mean, we could "put together" some related stuff

Small example

Please, tell us a little bit about yourself (* are mandatory)
> Name
> Country
> Birthday
....

Profile Picture 

Tell to the Community using your own words who you  are in ANY language you want 
======

[page 2]

Languages you're proficient 

Native language:
Strongest language (if applicable) 

{{A "add more languages" button}}


====

[[page 3]]

You've set up Tatoeba. What would you like to do next?

Add sentences
Translate sentences from ... into ....
etc

**Comment by jiru** [comment]
@RyckRichards Thanks for you suggestion!

@trang I realize I may have to ajaxify the profile edition page if I want to make it fit in the tutorial. Do you have plans to do that already?

**Comment by trang** [comment]
No, I don't have plans to ajaxify the profile edition. I mean eventually yes, but this is not in the priorities for the responsive UI. You can go ahead and do that.

The changes I will have to do in the profile page is mostly about how to deal with the info in the sidebar in the responsive layout.

**Comment by jiru** [comment]
I’ve finally created a wireframe to show you what I have in mind: https://downloads.tatoeba.org/wireframes/Register.html
I plan to use [Angular’s steppers component](https://material.angular.io/components/stepper/overview).

The final page show links that are yet to be defined. For the "Meet members" link, I’m thinking about something that shows active members having the same mother tongue or the same profile languages. The need for such page is showed in [UX test 1](https://en.wiki.tatoeba.org/articles/show/ux-test-1).

**Comment by jiru** [comment]
I designed the "Add sentences" page: https://downloads.tatoeba.org/wireframes/Add%20sentence.html

It’s basically our existing "Add sentences" page with embedded help. My idea is to initially replace the contents of the current "Add sentences" block with the help text when the user never added any sentences yet. After clicking the "Got it" button, a tooltip-like message on the top-right explains how to read the help again later. Clicking anywhere should hide the tooltip. This could be a pattern that we use elsewhere on the website to provide contextualized help. The last wireframe shows the help message displayed by clicking on the (?) button (the text is a little bit different).

Oh and I also added a kind of catchphrase on the top because the page looks very empty. :smile:

**Comment by ckjpn** [comment]
> Be creative. Avoid using the same words, names, topics, or patterns over and over again.

This sentence can probably be eliminated from a "first sentence" explanation.

It's also not a good guideline for the project in my opinion. I wonder who wrote this one.

Should I avoid using repeating words like "the," "he," "have," "out," etc. over and over again?

Should I avoid the creating sentences related to a topic I've already written about?
politics, weather, food, hotel, travel, etc.

Should I avoid using a sentence pattern I've already used? This would eliminate a lot of sentences, since languages are based on patterns.

Even the guideline about names is illogical.

Another valid view of this is that instead of adding common, everyday sentences with a unique name, use a wildcard name like Tom, so we get a large variety of sentence patterns, rather than a lot of the same sentence patterns with a lot of different names.

So instead of contributing sentences like all of these.

Tom made mistakes.
Mary made mistakes.
Sami made mistakes.
Layla made mistakes.
Mennad made mistakes.
Abner made mistakes.
Pawel made mistakes.
Yasser made mistakes.
Lars made mistakes.
Ahmed made mistakes.
Iskander made mistakes.
Karim made mistakes.
Omar made mistakes.
Esther made mistakes.

You could help eliminate this kind of pattern duplication by contributing your sentence with the name "Tom", or with "Mary" if a female name is required.

Tom made mistakes.
https://tatoeba.org/eng/sentences/search?from=eng&query=Tom+made+mistakes
See number of similar, related sentence that appear with this kind of search to understand the value.

This also means there is a chance that several members have already added sentences in their languages that might be able to be linked to your sentence.

**Comment by jiru** [comment]
About the "be creative" text, for your information this has always been displayed on the sidebar of the *Add sentences* page.

While I understand that the part about avoiding repetitions is controversial, do you agree that creativity should be encouraged? If so, what could be the sentence that goes after "Be creative."?

**Comment by ckjpn** [comment]
> About the "be creative" text, for your information this has always been displayed....

I'm fairly certain this message has gone up fairly recently.
The older message just said something about punctuation and capitalization, and possibly something about not using copyrighted stuff, but I don't think that was there either

**Comment by ckjpn** [comment]
> Do you agree that creativity should be encouraged? 

If this is for the first sentence, I think it's an unnecessary instruction.

It seems to imply that you expect that people joining that project are going to be adding a boring sentence as their first sentence.

Also, if you are going to say something about "annotation," that probably needs to be explained, so I would also suggest leaving that out.  Simple and to the point is better and more likely to be read and comprehended.

**Comment by jiru** [comment]
About the message on the sidebar, you are right. The code history indicates that part has been added by Trang last year. The statement about repeating names is controversial but remember that both views are accepted. What you copy-pasted is only one of the two opinions.

> If this is for the first sentence, I think it's an unnecessary instruction.
>
> It seems to imply that you expect that people joining that project are going to be adding a boring sentence as their first sentence.

This wasn’t my intention. Leaving out the controversial part about repetitions, I think the point is that members should feel free to contribute any kind of sentence, even if it’s not what you expect to see on a textbook or on a newspaper. I imagine that a lot of people unfamiliar with the project may refrain themselves from contributing as freely as they could. For example, they may think that they should only write the standard variant of their language, only literary text, or avoid dialects used by lower social classes or minorities. However, as a matter of fact we do value diversity and welcome all kinds of contributions. So not only I think this is a relevant answer to the question "what kind of sentences should I add" but it’s also one of Tatoeba’s core values, so I think it’s a good place to mention it. Now I may not have chosen the best wording, but it’s no reason to remove the line altogether, I’m sure we can work something out.

From Trang’s text, I renamed "Unleash your creativity!" into "be creative" because it sounded a bit too exuberant. But as you said if there is any assumption about the opposite it would be "you are not creative", while in Trang’s version it would be "you are not unleashing your creativity" which is better. So how about...

`• Feel free to be creative. We welcome all kinds of contributions and value diversity.`

> Also, if you are going to say something about "annotation," that probably needs to be explained, so I would also suggest leaving that out. Simple and to the point is better and more likely to be read and comprehended.

Maybe yes. I didn’t go into details about annotations because I assumed most annotations come from translations, not original sentences. Could you confirm that this assumption is correct?

**Comment by ckjpn** [comment]
> I didn’t go into details about annotations because I assumed most annotations come from translations, not original sentences. Could you confirm that this assumption is correct?

I think that's likely, too, which happens often in Japanese language textbooks for English speakers.  (He/She/You/ etc.) swim(s).

So, I think it wouldn't be necessary for the "add sentence" thing, which is aimed at original sentences and not translations.

https://downloads.tatoeba.org/wireframes/Add%20sentence.html
Woohoo, you are about to add your first sentence! Please pay attention to the following. ....

**Comment by jiru** [comment]
I created a [new version of Add sentence page for new users](https://downloads.tatoeba.org/wireframes/Add%20sentence.2.html) with rephrased and shorter text and inspiration from [Alan’s post](https://tatoeba.org/fra/wall/show_message/34798).

@trang Can I have your feedback on this, as well as the [new registration tutorial](#issuecomment-616238040)?

**Comment by alanfgh** [comment]
gillux, you wrote this:

Pay attention to the details, such as punctuation, capitalization and comma
> splice.
>

Since avoiding comma splices is included in "Pay attention to...
punctuation", I don't think we need to mention it specifically. Thus,I
think it would be better to write "... such as punctuation and
capitalization", or "... such as punctuation, capitalization, and
spelling." Nice as it would be to get people to avoid comma splices in
particular,
(a) many people are not familiar with the term (and linking to a definition
feels like overkill); the term "run-on sentences" was what I heard in
school, though it's less precise
(b) some people are not even familiar with the concept, and it takes a
while to teach
(c) avoiding comma splices is largely specific to English; in other
languages, comma splices are legal

Thanks,
Alan

**Comment by agrodet** [comment]
Just a side comment about unleashing one's creativity. If it was only up to me, it we would be written in bold, bigger rainbow font with triple arrows :) So I guess an average encouragement would be a good compromise.

More seriously, my personal experience with new people adding new sentences to Tatoeba has always been something like "You can add any sentence you'd like." "Really? But what about this? People may think it's strange, no?" and a following discussion about why it's not, quite the opposite. Long story short, some people, may it be for cultural reasons or others, are afraid of what other people will think of their contributions / them. For some of them, it is actually justified since they are contacted out of the blue to do this or that in this or that way, or encouraged to not do this or that.

**Comment by jiru** [comment]
@agrodet Thanks for your comment. I actually had the same experience with a new contributor I introduced to Tatoeba. This comforts me into the idea that the part about creativity (or whatever phrasing that would encourage a broad spectrum of contributions) is relevant. I think a longer explanation could even go into the FAQ.

On a side note, since we are touching at some quite sensitive texts, I’d be happy to have @Trang’s opinion.

**Comment by alanfgh** [comment]
> Woohoo, you are about to add your first sentence! Please pay attention to the following.
> 
> What kind of sentences should I add?
> 
> • Write sentences that sound natural, without forgetting punctuation and capitalization.
> • Be creative. Avoid using the same words, names, topics, or patterns over and over again.
> • Do not add copyrighted sentences. Avoid copy-pasting sentences from other websites unless their content is compatible with the CC BY license.
> • Do not include annotations inside the sentence.
> 
> I did a mistake.
> 
> • No problem. You can either fix your sentence using the <edit> icon or delete it using the <delete> icon.

I'm not sure about the "Woohoo". It's quite slangy, and I'm not sure everyone (particularly non-natives) will understand it.  "Congratulations!" might be an alternative. Or you could simply remove it and start with "You are about to add your first sentence!"

Also, it would be "I made a mistake", not "I did a mistake."

**Comment by jiru** [comment]
Thank you for your feedback. I tried to incorporate everybody’s comment and I created a [new version](https://downloads.tatoeba.org/wireframes/Add%20sentence.3.html).

**Comment by alanfgh** [comment]
I like the new version for the most part, but I seriously disagree with most of the last item:

> Feel free to be creative, without fearing what people might think of you when reading your sentences. On Tatoeba, we welcome all kinds of contributions, including slang, literature, dialects etc.

I would stop after "Feel free to be creative." Do we actually have a problem with people being inappropriately inhibited by what others might think of them from reading their sentences? If anything, it's the other way around. Regarding slang and literary language, these are hard to get right (especially for non-native speakers) and even harder to correct. We have a policy of allowing them, but I don't actively want to solicit them. As for dialects, I'm fine with encouraging them **as languages**, where we can develop norms for spelling. But when you start mixing in other dialects of a language with the standard dialect, you get a mess. Again, it becomes very hard to correct. Tags are not of much help in that regard (and new contributors can't apply them, anyway).

Our collection of sentences has organically developed certain characteristics over time, and this makes it useful. I don't want us to tell people before they've even started here to contribute kinds of sentences that clash with the body of our corpus. They should get a feeling for the kind of sentences that we currently have. After a while, they can see for themselves what kind of sentences "push the envelope", how frequent they are, and how they are handled.

**Comment by RyckRichards** [comment]
Is there any way to redirect users which language is not on Tatoeba to leave a message to the Languages Team? Maybe some of them have to figure out and so, they won't do that.

Also, they can be warned if the UI they're using is not fully translated into their native language (or even doesn't exist one). Maybe something like; "We're working on having as many languages on the UI as possible. Some of them might be not fully translated yet. Would you like to join the translators' team?"

**Comment by trang** [comment]
> I created a [new version of Add sentence page for new users](https://downloads.tatoeba.org/wireframes/Add%20sentence.2.html) with rephrased and shorter text and inspiration from [Alan’s post](https://tatoeba.org/fra/wall/show_message/34798).
> 
> @trang Can I have your feedback on this, as well as the [new registration tutorial](#issuecomment-616238040)?

Here's my long-due feedback 🙂 

**For the post-registration steps**

I'm wondering if what you have in step (4) shouldn't actually be in step (1) instead. Or it could even be integrated into the homepage, under a section "What would you like to do today/now?", at the top of the page.

I think in general we should first figure out what the user wants to do, then based on that, we ask information necessary or relevant to the chosen activity.

For instance, if I choose "Proofread", then the step for self-introduction would not be very relevant. On the other hand, if the user chooses "Meet members", then prompting them to write a self-introduction makes sense.

On a more technical level, note that AngularJS Material doesn't have a stepper component. That's for Angular Material only.

**For the "Add sentence" page**

It seems the discussion about the wording is trying to solve the problem that some users have been afraid to create interesting sentences just because they think the sentences don't fit the "norms" of Tatoeba.

I'm not too convinced that changing the wording of our current "Unleash your creativity" will really make a big impact. I think it's completely normal that some people will be afraid at first. The best thing to do is just to give them time. They might create very "textbook-like" sentences at the beginning, but after a few weeks or months, as they get more comfortable with Tatoeba and its community, they will also feel more comfortable creating more original/authentic sentences.

Perhaps a more useful tip to give to those who are hesitant about what kind of sentences to add, is to explain what they can do in case they are not sure if a sentence is okay or not: they can ask on the Wall, read our guidelines on how to create good sentences, or just add the sentence and if no one complains then it's okay.

But in any case, I think reworking the "Add sentences" page is a whole issue on its own and would be a bit off-topic to solve as part of this issue. This issue here is more about ensuring that new users have a general understanding of what they can do on Tatoeba and where to get help if they need help.

**Comment by jiru** [comment]
@trang Alright so let’s focus on the registration page. :slightly_smiling_face: 

The 4th step definitely overlaps with the homepage as in #2169. But if I am to rework the home page (for logged-in users) too, I feel like I will have to deal complains and suggestions of all sorts and this issue won’t be finished before next year. :sweat_smile: Or do you think inserting the 4th step block on the top of the homepage for logged-in users, just like that, would be okay? Or how about making it a separate page people could browse again afterwards, something like `/activities/index`?

> On a more technical level, note that AngularJS Material doesn't have a stepper component. That's for Angular Material only.

You are right. But it looks like there is also an implementation for AngularJs: https://www.npmjs.com/package/md-steppers

**Comment by jiru** [comment]
Oh and I agree with:

> For instance, if I choose "Proofread", then the step for self-introduction would not be very relevant. On the other hand, if the user chooses "Meet members", then prompting them to write a self-introduction makes sense.

How about setting up a kind of "consider filling up your profile" suggestion popup that would be triggered by descriptionless profiles on various places where the user is to interact with the community, such as sentence comments, wall posts, private messages etc.

**Comment by trang** [comment]
> Or do you think inserting the 4th step block on the top of the homepage for logged-in users, just like that, would be okay?

I think the section could be something that the user can hide once they no longer need it. And we could make it not show up at all for the existing users. Only for the new users.

![Screenshot_2020-08-19 Tatoeba Collection of sentences and translations](https://user-images.githubusercontent.com/221850/90663586-2554e780-e24a-11ea-8838-0ede3bcb67bd.png)


> Or how about making it a separate page people could browse again afterwards, something like `/activities/index`?

It's also a possibility but some users may not easily find their way back to that separate page. Or at least not as easily as the homepage.

**Comment by jiru** [comment]
I see. I’m afraid this would be bring more confusion however, because users tend to *stick to the navigation paths they initially discovered*. (I’m no UX expert, this is just something I’ve personally observed in UX tests and by observing users in general.) The way *you* see it is that the top menu is the "normal" way to reach pages while this new homepage block is a kind of "temporary shortcut for newcomers". This is because you initially learned to use Tatoeba with the top menu. However, if newcomers are to initially learn to reach, say, the "Add sentence" page from the home page, then it will be *their* "normal" way. So it would make no sense for them to click the "don't show any more" button.

I’m not sure what’s the best way to handle such situation.

**Comment by trang** [comment]
Then when clicking on "Add sentence", could explain how to reach the "Add sentence" page instead of redirecting to that page. Same for the other activities. Once new users have learned how to access the main areas of the website, then it will make sense for them to hide this section.

Otherwise, if we think it would actually be beneficial to introduce these new navigation paths on the homepage, then we just add an option to hide this section (like what we have for the random sentence) instead of having a button to hide it.
